invenio_rdm_records/records/mappings/v7/rdmrecords/drafts/draft-v1.0.0.json

```diff
@@ -176,21 +176,24 @@
           },
           "locations" : {
             "properties" : {
-              "description" : {
-                "type" : "text"
+              "geometry": {
+                "properties": {
+                  "type": {
+                    "type": "keyword"
+                  },
+                  "coordinates": {
+                    "type": "float"
+                  }
+                }
               },
               "place" : {
                 "type" : "text"
```

Comment:
Question: Should ``place`` be text or keyword?

Reply:
My thought was that we want some analyzers and queries with certain levenstain distance. But if exact match is enough, I'll change to `keyword`.

Reply:
Let's wait and see how until we know how the search behaves.